MatrixSDK/MXSession.m

```diff
@@ -2902,6 +2936,18 @@ - (MXHTTPOperation*)eventWithEventId:(NSString*)eventId
                              success:(void (^)(MXEvent *event))success
                              failure:(void (^)(NSError *error))failure
 {
+    if (eventId == nil)
+    {
+        if (failure)
+        {
+            MXError *error = [[MXError alloc] initWithErrorCode:kMXErrCodeStringNotFound
```

Comment:
Can we dump the call stack (`[NSThread callStackSymbols]`) to find the culprit more easily?

Reply:
Yeah i thought about it too, but later i’ve decided to add individual logs for every caller. Do you think we still need it?